rosetta-source/src/main/rosetta/event-common-type.rosetta

```diff
@@ -571,12 +571,12 @@ type TransferBase:
     identifier Identifier (0..*) <"Represents a unique reference to the transfer.">
         [metadata scheme]
     quantity NonNegativeQuantity (1..1) <"Represents the amount of the asset to be transferred.">
-    observable Observable (0..1) <"Represents the object that is subject to the transfer, it could be an asset or a reference.">
+    asset Asset (0..1) <"Represents the object that is subject to the transfer, it could be an asset or a reference.">
```

Comment:
Make `asset` mandatory. It was only optional before because `observable` was the wrong attribute type, hence in the "cash" scenario there was none.